api/tests/test_data/extra_test_data.json

```diff
@@ -1,4 +1,117 @@
 {
+  "feeds": [
```

Comment:
The reason there was no feeds section in this file is because the feeds should be added to the csv file.

Reply:
Yea i understand that -- however there's no easy way to add feeds with multiple locations from sources_test.csv

Reply:
Understood.
This data should probably go into tests/integration/test_data/extra_test_data.json since it concerns only the integration directory.